project/CLI.py

```diff
@@ -19,8 +19,17 @@ def __login(self, args):
             return "Login failed"
         return "Login successful"
 
+    def __create_game(self, args):
+        try:
+            self.gm = self.game.start(args[1])
+        except IndexError:
+            return "Invalid Parameters"
+        if self.is_gm is False:
```

Comment:
This check needs to be before you make any changes (before attempting to set self.game